07/ID_tryExcept

```diff
@@ -0,0 +1,52 @@
+# Input ID number in Czech Republic
```

Comment:
Chceš aby som ti robila review aj na toto alebo už len na tie samostatné súbory?

Reply:
no tohle,...to fakt nevím, jak se mohlo stát :-D :-D  mrknu na to

Reply:
:) už jsem to snad opravila všechno